pom.xml

```diff
@@ -1336,37 +1345,14 @@
           <artifactId>docker-maven-plugin</artifactId>
           <version>${version.docker.plugin}</version>
         </plugin>
-        <plugin>
-          <groupId>io.vertx</groupId>
```

Comment:
Why do you remove the vert.x stuff here?

Reply:
Basically, it's a relic 🦕.

That plugin is no longer used since older versions of Vert.x and this was just hanging around doing nothing! 

The `maven-shade-plugin` Maven plugin now to create the Uberjar 👍. In fact, the latest version of that plugin was released in 2015 and nobody uses it any more - https://mvnrepository.com/artifact/io.vertx/vertx-maven-plugin .

Reply:
Okay, then it is great to get rid of it :)